### PR TITLE
cloud: add release notes

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,53 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## April 25, 2023
+
+**General changes**
+
+- For the first five Serverless Tier clusters in your organization, TiDB Cloud provides a free usage quota for each of them as follows:
+
+    - Row storage: 5 GiB
+    - [Request Units](/tidb-cloud/tidb-cloud-glossary.md#request-unit) (RUs): 50 million RUs per month
```
Comment (lilin90) [inline tidb-cloud/tidb-cloud-release-notes.md:18]: It seems that this anchor link does not work well.

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -8,6 +8,53 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    Until May 30, 2023, Serverless Tier clusters are still free, with a 100% discount off. After that, usage beyond the free quota will be charged.
+
+    You can easily [monitor your cluster usage or increase your usage quota](/tidb-cloud/manage-serverless-spend-limit.md#manage-spend-limit-for-serverless-tier-clusters) in the **Usage This Month** area of your cluster **Overview** page. Once the free quota of a cluster is reached, the read and write operations on this cluster will be throttled until you increase the quota or the usage is reset upon the start of a new month.
+
+    For more information about the RU consumption of different resources (including read, write, SQL CPU, and network egress), the pricing details, and the throttled information, see [TiDB Cloud Serverless Tier Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details).
```
Comment (lilin90) [inline tidb-cloud/tidb-cloud-release-notes.md:24]: Wondering whether we need to keep the "egress" word here. 🤔

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 3]
```diff
@@ -8,6 +8,53 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+- Upgrade the default TiDB version of new [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters from [v6.5.1](https://docs.pingcap.com/tidb/v6.5/release-6.5.1) to [v6.5.2](https://docs.pingcap.com/tidb/v6.5/release-6.5.2).
+
+- Provide a maintenance window feature to enable you to easily schedule and manage planned maintenance activities for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters.
+
+    A maintenance window is a designated timeframe during which planned maintenance activities, such as operating system updates, security patches, and infrastructure upgrades, are performed automatically to ensure the reliability, security, and performance of the TiDB Cloud service.
+
+    During a maintenance window, although there might be temporary connection disruptions or QPS fluctuations, the clusters remain available, and SQL operations, the existing data import, backup, restore, migration, and replication tasks can still run normally. See [a list of allowed and disallowed operations](/tidb-cloud/configure-maintenance-window.md#allowed-and-disallowed-operations-during-a-maintenance-window) during maintenance.
```
Comment (lilin90) [inline tidb-cloud/tidb-cloud-release-notes.md:36]:
```suggestion
    During a maintenance window, although temporary connection disruptions or QPS fluctuations might occur, the clusters remain available, and SQL operations, the existing data import, backup, restore, migration, and replication tasks can still run normally. See [a list of allowed and disallowed operations](/tidb-cloud/configure-maintenance-window.md#allowed-and-disallowed-operations-during-a-maintenance-window) during maintenance.
```

File: tidb-cloud/tidb-cloud-release-notes.md (hunk 1 above)
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:18]:
```suggestion
    - [Request Units (RUs)](/tidb-cloud/tidb-cloud-glossary.md#request-unit): 50 million RUs per month
```

Comment (qiancai) [reply tidb-cloud/tidb-cloud-release-notes.md:18]: It is is a new term to be added in https://github.com/pingcap/docs/pull/13335/files.

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 4]
```diff
@@ -8,6 +8,53 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    We will strive to minimize the frequency of maintenance. If a maintenance window is planned, the default start time is 03:00 Wednesday (based on the time zone of your TiDB Cloud organization) of the target week. To avoid potential disruptions, it is important to be aware of the maintenance schedules and plan your operations accordingly.
+
+    - To keep you informed, TiDB Cloud will send you three email notifications for every maintenance window: one before, one starting, and one after the maintenance tasks.
+    - To minimize the maintenance impact, you can modify the maintenance start time to your preferred time or defer maintenance activities on the **Maintenance** page.
+
+    For more information, see [Configure Maintenance Window](tidb-cloud/configure-maintenance-window.md).
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:43]:
```suggestion
  For more information, see [Configure maintenance window](tidb-cloud/configure-maintenance-window.md).
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 5]
```diff
@@ -8,6 +8,53 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+- Improve load balancing of TiDB and reduce connection drops when you scale TiDB nodes of [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters hosted on AWS.
+
+    - Support automatically migrating existing connections to new TiDB nodes when you scale out TiDB nodes.
+    - Support automatically migrating existing connections to available TiDB nodes when you scale in TiDB nodes.
+
+    Currently, this feature is provided for all Dedicated Tier clusters hosted on AWS.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:50]:
```suggestion
  Currently, this feature is provided for all Dedicated Tier clusters hosted on AWS.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 6]
```diff
@@ -8,6 +8,53 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+## April 25, 2023
+
+**General changes**
+
+- For the first five Serverless Tier clusters in your organization, TiDB Cloud provides a free usage quota for each of them as follows:
+
+    - Row storage: 5 GiB
+    - [Request Units (RUs)](/tidb-cloud/tidb-cloud-glossary.md#request-unit): 50 million RUs per month
+
+    Until May 30, 2023, Serverless Tier clusters are still free, with a 100% discount off. After that, usage beyond the free quota will be charged.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:20]:
```suggestion
  Until May 30, 2023, Serverless Tier clusters are still free, with a 100% discount off. After that, usage beyond the free quota will be charged.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 7]
```diff
@@ -8,6 +8,53 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## April 25, 2023
+
+**General changes**
+
+- For the first five Serverless Tier clusters in your organization, TiDB Cloud provides a free usage quota for each of them as follows:
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:15]:
```suggestion
- For the first five [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters in your organization, TiDB Cloud provides a free usage quota for each of them as follows:
```

File: tidb-cloud/tidb-cloud-release-notes.md (hunk 6 above)
Comment (lilin90) [reply tidb-cloud/tidb-cloud-release-notes.md:20]: It seems that the original indentation works well?

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by lilin90 — APPROVED** [review]
Rest LGTM

File: tidb-cloud/tidb-cloud-release-notes.md (hunk 6 above)
Comment (qiancai) [reply tidb-cloud/tidb-cloud-release-notes.md]: The original indentation won't work well in the doc website because there is an indented list before it😂

**Pull request merged**